content/en/docs/reference/glossary/master.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Master
+id: master
+date: 2020-04-16
+short_description: >
+  Deprecated term, used as synonym for nodes running the control-plane.
```

Comment:
nit:
```suggestion
  Deprecated term, used as synonym for nodes running the control plane.
```
“The control plane” is idiomatic in English and more widely used than “the control-plane”. If using control-plane as an adjective there can be stronger grounds for replacing the space.

Reply:
I didn't know that! Fixed.